src/components/Todolist/Todolist.tsx

```diff
@@ -0,0 +1,42 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filteredTodos: Todo[] | null;
+};
+
+export const Todolist: React.FC<Props> = ({ filteredTodos }) => {
+  return (
+    <section className="todoapp__main">
+      {filteredTodos?.map(todo => (
+        <div
+          className={cn('todo', {
+            completed: todo.completed,
+          })}
+        >
+          <label className="todo__status-label">
+            <input
+              type="checkbox"
+              className="todo__status"
+              checked
+            />
+          </label>
+
+          <span className="todo__title">{todo.title}</span>
+
+          <button
+            type="button"
+            className="todo__remove"
+          >
+            ×
+          </button>
+
+          <div className="modal overlay">
+            <div className="modal-background has-background-white-ter" />
+            <div className="loader" />
+          </div>
+        </div>
+      ))}
```

Comment:
Haven't added a key and maybe it would be better to make a separate component for todo info